.ci/deploy-linux.sh

```diff
@@ -20,37 +23,34 @@ if [ "$DEPLOY_APPIMAGE" = "true" ]; then
 	curl -sL https://github.com/RPCS3/AppImageKit-checkrt/releases/download/continuous2/AppRun-patched-x86_64 -o ./appdir/AppRun
 	chmod a+x ./appdir/AppRun
 	curl -sL https://github.com/RPCS3/AppImageKit-checkrt/releases/download/continuous2/exec-x86_64.so -o ./appdir/usr/optional/exec.so
-	
+
 	# Compile checker binary for AppImageKit-checkrt
-	
+
 	# This may need updating if you update the compiler or rpcs3 uses newer c++ features
 	# See https://github.com/gcc-mirror/gcc/blob/master/libstdc%2B%2B-v3/config/abi/pre/gnu.ver
 	# for which definitions correlate to which CXXABI version.
 	# Currently we target a minimum of GLIBCXX_3.4.26 and CXXABI_1.3.11
 	printf "#include <bits/stdc++.h>\nint main(){std::make_exception_ptr(0);std::pmr::get_default_resource();}" | $CXX -x c++ -std=c++2a -o ./appdir/usr/optional/checker -
-	
+
 	# Package it up and send it off
 	./squashfs-root/usr/bin/appimagetool /rpcs3/build/appdir
 	ls
-	COMM_TAG="$(grep 'version{.*}' ../rpcs3/rpcs3_version.cpp | awk -F[{,] '{printf "%d.%d.%d", $2, $3, $4}')"
+
+	COMM_TAG="$(grep 'version{.*}' ../rpcs3/rpcs3_version.cpp | awk -F[\{,] '{printf "%d.%d.%d", $2, $3, $4}')"
 	COMM_COUNT="$(git rev-list --count HEAD)"
+	COMM_HASH="$(git rev-parse --short=8 HEAD)"
+	RPCS3_APPIMAGE="rpcs3-v${COMM_TAG}-${COMM_COUNT}-${COMM_HASH}_linux64.AppImage"
+
 	curl -sLO https://github.com/hcorion/uploadtool/raw/master/upload.sh
-	
-	if [[ -n "$BUILD_SOURCEVERSION" ]]; then
-		COMMIT_HASH=$BUILD_SOURCEVERSION
-	elif [[ -n "$TRAVIS_COMMIT" ]]; then
-		COMMIT_HASH=$TRAVIS_COMMIT
```

Comment:
I can't remember why we don't just use the git rev-parse and instead use the TRAVIS_COMMIT variable, but I'm fairly certain there was a reason 🤔

Reply:
Yeah, there's a lot going on with the numbering system. I think `git describe` provides better info than what we do, but I don't want to change it since I'm not sure of the reasoning behind the current system.

Reply:
Tracking down git blame it looks like it was introduced in the original by Ani in https://github.com/RPCS3/rpcs3/pull/2830.
@AniLeo any idea why you used `TRAVIS_COMMIT`?

Reply:
No reason, it was already defined and that thing was only for Travis at the time
As long as the end result is the same I don't think it matters